Modules/_elementtree.c

```diff
@@ -1242,12 +1254,18 @@ _elementtree.Element.find
     path: object
     namespaces: object = None
 
+Find first matching element by tag name or path.
+
+*path* is a string having either an element tag or an XPath,
+*namespaces* is an optional mapping from namespace prefix to full name.
```

Comment:
Consider using per-parameter docstrings here (see the [Argument Clinic Howto](https://devguide.python.org/development-tools/clinic/) for inspiration).

Consider this also for the rest of the PR.